docs/pages/desktop-access/reference/configuration.mdx

```diff
@@ -20,6 +20,10 @@ The Windows Desktop Service can be deployed in two modes.
 
 ### Direct mode
 
+<Admonition type="note">
```

Comment:
I would move this down right after the sentence that says "requires direct connection to the auth server" and then be more specific. (When we host your auth sever it is not internet accessible)

Reply:
Please see update.